src/LineUI/LineUnit.tsx

```diff
@@ -75,6 +76,10 @@ const GrabHandleGroup = styled.g<{ showIndex: boolean, originalRadius: number, s
   `};
 `;
 
+const Circle = styled.circle`
```

Comment:
Can we give this a more descriptive name rather than just the element name. If it's for the direction mark background then perhaps just `DMCircle` is suitable.

Reply:
Done